exportoptions/properties.go

```diff
@@ -119,6 +141,20 @@ func ParseMethod(method string) (Method, error) {
 	}
 }
 
+// UpgradeExportMethod replaces the legacy export method strings with the ones available in Xcode 15.3 and later.
+func UpgradeExportMethod(method Method) Method {
```

Comment:
What about adding the Xcode version to the method name? I can imagine ore changes to the method names in the future.
Something like `UpgradeToXcode15_3MethodNames`

Reply:
Renamed.